core/pom.xml

```diff
@@ -638,6 +640,13 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>make-jni-headers</id>
+            <properties>
+                <makeJniHeaderArg1>-h</makeJniHeaderArg1>
+                <makeJniHeaderArg2>${project.build.directory}/headers</makeJniHeaderArg2>
+            </properties>
+        </profile>
```

Comment:
This is useful when doing JNI work:
Running `mvn compile -P make-jni-headers` will create a `target/headers` directory containing all the right signatures, thus cutting down on guesswork.

For calling back into Java I've found `javap -s path/to/Some.class` pretty useful too.